build.gradle

```diff
@@ -472,6 +473,24 @@ project('spring-integration-cassandra') {
     }
 }
 
+project('spring-integration-cloudevents') {
+    description = 'Spring Integration Cloud Events Support'
+
+    dependencies {
+        api "io.cloudevents:cloudevents-core:$cloudEventsVersion"
+        optionalApi "io.cloudevents:cloudevents-json-jackson:$cloudEventsVersion"
+
+        optionalApi("io.cloudevents:cloudevents-avro-compact:$cloudEventsVersion") {
+            exclude group: 'org.apache.avro', module: 'avro'
+        }
+        optionalApi "org.apache.avro:avro:$avroVersion"
+        optionalApi "io.cloudevents:cloudevents-xml:$cloudEventsVersion"
```

Comment:
Are these all `optional` deps really necessary for our production code?
Maybe `test` scope instead?